xc/xc7/tests/soc/litex/mini_ddr/CMakeLists.txt

```diff
@@ -17,6 +18,7 @@ add_fpga_target(
     minilitex_ddr_arty.v
     VexRiscv_Lite.v
   INPUT_XDC_FILE minilitex_ddr_arty.xdc
+  INPUT_SDC_FILE arty.input.sdc
```

Comment:
What is the difference between `INPUT_SDC_FILE` and `SDC_FILE`?

Reply:
INPUT_SDC_FILE contains a SDC command given by the user, e.g. with a single `create_clock` command, the SDC_FILE is the one generated by Yosys. I kept it for backward compatibility with other existing test cases that do not use the plugin yet. In the future there should be no need for the generated SDC, so SDC_FILE in this case.

Reply:
Why does the generated file (e.g. `SDC_FILE`) need to be exposed to the user at all?

Reply:
It doesn't. This was just a way of introducing the change without affecting the rest of the tests which would still use the "old" flow. However, if the plugin works then if the input SDC already contains all necessary clocks then the output sdc shouldn't differ, hence there should be no impact on the test.
I made the changes to have VPR use the generated SDC.

Reply:
This is fixed. But the PR still depends on the SDC plugin.